cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -86,6 +87,69 @@ Instead they are defined in terms of their underlying _privileges_ and they can
 In addition, any new roles created can by assigned any combination of _privileges_ to create the specific access control desired.
 A major additional capability is _sub-graph_ access control whereby read-access to the graph can be limited to specific combinations of label, relationship-type and property.
 
+[[administration-security-syntax-rules]]
+== Syntax summaries
+Almost all administration commands have variations in the commands. Parts of the command that are optional or can have multiple values are most common.
+To show all versions of a command, a summary of the syntax will be presented. These summaries will use some special characters to indicate such variations.
+
+The special characters and their meaning are as follows:
+
+.Special characters in syntax summeries
+[options="header", width="100%", cols="1a,3,3"]
+|===
+| Character | Meaning | Example
+
+| `\|`
+| Or, used to indicate alternative parts of a command. Needs to be part of a grouping.
+| If the syntax needs to specify either a name or `*`, this can be indicated with `name \| *`.
+
+| `{` and `}`
+| Used to group parts of the command, common together with `\|`.
+| To use the `or` in the syntax summary it needs to be in a group, `{name \| *}`.
+
+| `[` and `]`
+| Used to indicate an optional part of the command. It also groups alternatives together, when there can be either of the alternatives or nothing.
+| If a keyword in the syntax can either be in singular or plural, we can indicate that the `S` is optional with `GRAPH[S]`.
+
+| `...`
+| Repeated pattern, the command part immediately before this is repeated.
+| A comma separated list of names would be `name[, ...]`.
+
+| `"`
+| When a special character is part of the syntax itself, we surround it with `"` to indicate this.
```

Comment:
In general I agree that we must come up with a way to distinguish between special characters that have a specific meaning for syntax like optional and special characters that is actually part of the command. I'm not sure if this is the best way to do it though. At least I would like `\` better as an escape character than `"`. Maybe someone in docs team have an idea?

Reply:
I think we got the `"` from PRDs, and it is used in 4.1 for the properties in the [READ privilege](https://neo4j.com/docs/cypher-manual/current/administration/security/reads/#administration-security-reads-read) which is why I have that here

I just documented what we were using :P Though I agree that it isn't the nicest

Reply:
If we decide to change it to something else we need to update both where it already exists and some of my currently open PRs :P